blender-mesh/src/lib.rs

```diff
@@ -93,12 +92,20 @@ impl BlenderMesh {
     }
 
     /// A map of material name to the material's data
-    pub fn materials(&self) -> &HashMap<String, PrincipledBSDF> {
+    pub fn materials(&self) -> HashMap<String, PrincipledBSDF> {
```

Comment:
Can make this

```
pub fn material_with_name(&self, name: &str) -> Option<&PrincipledBSDF>;
```

and then just iterate the vec to return the name. I'd imagine that ~all models have a small enough number of materials that iterating `O(N)` on the vec is fine.